packages/redux-devtools-inspector-monitor/src/tabs/StateTab.tsx

```diff
@@ -16,18 +18,39 @@ const StateTab: React.FunctionComponent<
   labelRenderer,
   dataTypeKey,
   isWideLayout,
-}) => (
-  <JSONTree
-    labelRenderer={labelRenderer}
-    theme={getJsonTreeTheme(base16Theme)}
-    data={nextState}
-    getItemString={(type, data) =>
-      getItemString(styling, type, data, dataTypeKey, isWideLayout)
-    }
-    invertTheme={invertTheme}
-    hideRoot
-  />
-);
+}) => {
+  const [sortObjectKeys, setSortObjectKeys] = useState(false);
+  const [disableCollection, setDisableCollection] = useState(false);
+
+  useEffect(() => {
+    if (!chrome || !chrome.storage) return;
+    const storage = isFF
+      ? chrome.storage.local
+      : chrome.storage.sync || chrome.storage.local;
+    storage.get(
+      ['sortStateTreeAlphabetically', 'disableStateTreeCollection'],
+      function (result) {
+        setSortObjectKeys(!!result.sortStateTreeAlphabetically);
+        setDisableCollection(!!result.disableStateTreeCollection);
+      }
+    );
+  }, []);
+
+  return (
+    <JSONTree
+      labelRenderer={labelRenderer}
+      theme={getJsonTreeTheme(base16Theme)}
+      data={nextState}
+      getItemString={(type, data) =>
+        getItemString(styling, type, data, dataTypeKey, isWideLayout)
+      }
+      invertTheme={invertTheme}
+      hideRoot
+      sortObjectKeys={sortObjectKeys}
+      {...(disableCollection ? { collectionLimit: 0 } : {})}
```

Comment:
Can you explain what the use case for this option is?

Reply:
If the state does have a lot of keys they are collapsed. This option prevents this.
> `collectionLimit: number` - sets the number of nodes that will be rendered in a collection before rendering them in collapsed ranges